app/views/layouts/application.html.erb

```diff
@@ -23,7 +23,7 @@
   <%= render partial: 'shared/google_tag_manager_body' %>
 
   <div class="th-wrapper">
-    <header id="earthdata-tophat2" />
+    <header id="earthdata-tophat2"></header>
```

Comment:
good catch, while this may have been fine under the xhtml days and is allowed in html5, I think this syntax is more clear.